db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
+          description: Dont scan image-inspectors image
+          modifier: deny
+          expression: !ruby/object:MiqExpression
+            exp:
+              ENDS WITH:
+                field: ContainerImage-name
+                value: "image-inspector"
+            context_type:
+          towhat: ContainerImage
+          file_mtime:
+          guid: e744245a-3d08-11e6-8d39-02422087d789
```

Comment:
I created the basic template using the UI and then edited that. I hope that this guid that I got from the UI is still good. Can anyone verify this? @moolitayer @simon3z